### DISCUSSION TITLE
New version doesn't capture TS files anymore (fixed in v8.0.18)

### DISCUSSION THREAD

**Discussion by miro011** [subhra74/xdm#962] [General]
I've encountered lots of sites that use .ts streams. The previous version (7.x.x) was able to mostly capture them. The 8.x.x versions however can not.

That's after i added "TS" to "XDM will show download option when below video formats...." in the Browser Monitoring settings.

Example site: https://zoechip.cc

**Comment by subhra74** [comment]
Please try latest version 8.0.18, and let me know if this solves the issue. Please note after installing the new version you need to go to browser integration page and click on appropriate browser integration button to fix the browser integration issues
https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.18-beta

**Reply by miro011** [reply]
It appears to be working. Thanks a lot!

**Comment by roachboy** [comment]
The archive can be downloaded, but the video cannot be played and an error message is displayed.
Video format is TS.
The previous version (7.x.x) was able to mostly capture them. The 8.x.x versions however can not.

Example site: https://jable.tv/

**Reply by miro011** [reply]
Using the 8.0.18 version, that site you gave works too. I just tested it. Once TS is detected it should be detected everywhere. It's not on "per-site" basis from what I understand.

**Reply by roachboy** [reply]
It can be downloaded, but the video cannot be played
The software is potplayer 1.7

**Reply by miro011** [reply]
> It can be downloaded, but the video cannot be played The software is potplayer 1.7

Well, install other software then. There are also video converters programs available if you insist on using Potplayer. Otherwise, VLC and from what you're saying Potplayer don't work well with TS files. Also, if I remember correctly, you could convert ts to mp4 in VLC.

Other than that, SMPlayer (mpv), which is what I use on Linux right now works perfectly fine. It's also available on Windows. On Windows there is also GomPlayer and KmPlayer that can do this. And there are probably other players on both platforms that can play TS files that I don't know about.

**Reply by roachboy** [reply]
The same file, I downloaded it with 7.xx and it can be played normally.
Therefore, it is provided as a reference for software developers. Is this a BUG?
Thank you for your detailed answer.
